firebase_admin/db.py

```diff
@@ -199,6 +235,35 @@ def delete(self):
         """
         self._client.request_oneway('delete', self._add_suffix())
 
+    def transaction(self, transaction_update):
+        """Write to database using a transaction.
+
+        Args:
+            transaction_update: function that takes in current database data as a parameter.
+
+        Returns:
+            bool: True if transaction is successful, otherwise False.
+
+        Raises:
+            ValueError: If transaction_update is not a function.
+
+        """
+        if not callable(transaction_update):
+            raise ValueError('transaction_update must be a function.')
+
+        tries = 0
+        etag, data = self._get_with_etag()
+        val = transaction_update(data)
+        while tries < _TRANSACTION_MAX_RETRIES:
+            success, etag, snapshot = self._update_with_etag(val, etag)
```

Comment:
Good point. It seems Node and Java SDKs support a similar feature. In Node the update function can return `undefined` to abort, and in Java there's a `Transaction.abort()` method. In any case lets hold off on implementing this change. I'll run this by the API review team, and update this thread with the outcome.